chessgame/src/common/pieces/Pawn.ts

```diff
@@ -0,0 +1,32 @@
+import Piece from "./_Interface";
+
+class Pawn extends Piece {
+    type: string = "Pawn";
+
+    canMove(column: string, row: number): boolean {
+        if(this.column != column){
+            return false;
+        }
+
+        if (this.isWhite){
```

Comment:
Tu mozna uladnic delikatnie, ze` colorMultiplier = isWhite 1 ? -1` i np `this.row + colorMultiplier * 1 == row` i 8 linijek zaoszczedzonych xd